deploy.sh

```diff
@@ -89,8 +92,8 @@ fi
 rm -fr datafiles/db.sqlite3
 python3 manage.py migrate
 # for apache admin display
-python3 manage.py collectstatic || true
-python3 manage.py createsuperuser
+yes 'yes' | python3 manage.py collectstatic || true
+python3 manage.py createsuperuser --username admin --email example@example.com --noinput
```

Comment:
if there is a method that the created admin superuser could log in as well, it would be good, not sure if there is any work around for this?

Reply:
I don't understand the question.

Reply:
from the help of the createsuperuser command, we could see the following description:
    --noinput, --no-input
                        Tells Django to NOT prompt the user for input of any
                        kind. You must use --username with --noinput, along
                        with an option for any other required field.
                        Superusers created with --noinput will not be able to
                        log in until they're given a valid password.

so need another step to set the password for the admin user to be able to login via web GUI.